main.go

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright (c) 2019-2021 Red Hat, Inc.
```

Comment:
Not sure if it matters, but should this be:
```suggestion
// Copyright (c) 2021 Red Hat, Inc.
```

Reply:
:+1: Also added copyright headers to the rest of the go files.